src/modules/Delete.js

```diff
@@ -0,0 +1,19 @@
+const deleteTask = (id) => {
+  const listDisplay = document.querySelector('.display-list');
+  let allTasks = JSON.parse(localStorage.getItem('todo'));
+  const btn = document.getElementById(`rmv-${id}`);
+  allTasks = allTasks.filter((task) => task.index !== id);
+
+  allTasks.forEach((i, j) => {
+    i.index = j;
+  });
+
+  allTasks = localStorage.setItem('todo', JSON.stringify(allTasks));
+  btn.previousSibling.parentElement.parentElement.remove();
+
+  if (allTasks.length === 0) {
+    listDisplay.innerHTML = '<hr/><p>No tasks are available now!</p>';
+  }
+};
+
+export default deleteTask;
```

Comment:
Hi @aima98 
You have done a great job implementing the different features of this app but there is a little update to complete the task.

- [ ] Please your app throws an error every time a user clicks the side button to delete a to-do in the list.
**Take A Look Blow**
![delete](https://user-images.githubusercontent.com/55596970/228727643-603eeeef-16cc-427f-a085-6269c0eb295b.png)
**Please fix the above error**